### ISSUE TITLE
NormalizedRegionOfInterest2D.msg are not compatible with other ROS4HRI packages in the walkthrough

### ISSUE THREAD

**Issue by Luca-Pozzi** [ros4hri/pyhri#5]
Hi! 
First of all thank you for this repo and for the ROS4HRI project, impressive effort!

# Issue

I would like to report that, while following the [ROS4HRI walkthrough](http://wiki.ros.org/hri/Tutorials/ROS4HRI-walk-through), I run into this disclaimer:
> Do not use pyhri as packaged in noetic. This version is outdated.
> Instead, download the latest release from [github.com/ros4hri/pyhri](https://github.com/ros4hri/pyhri/), and install it manually (using eg catkin build)

Unfortunately, after cloning and building the latest `pyhri` release, I experienced an issue when running sections 1.5 and 1.6 (i.e. running the _face detector_, _engagement detector_, and _person manager_ together). The engagement detector (taken from [here](https://gitlab.com/pal-robotics-workshops/ros4hri_iros_2022_tutorial/-/blob/main/src/engagement_detector/engagement.py) with minimal modification -thank you for this script as well!-) kept failing with error:
```
Could not process inbound connection: topic types do not match: [hri_msgs/NormalizedRegionOfInterest2D] vs. [sensor_msgs/RegionOfInterest]
```

# What I have tried

## - Modify the output of [`hri_face_detect/detect`](https://github.com/ros4hri/hri_face_detect/blob/main/nodes/hri_face_detect/detect)  - NOT WORKING
I succeeded in making the `detect` node from `hri_face_detect` publish `hri_msgs/NormalizedRegionOfInterest2D` (which is quite simple), but kept experiencing the same issue as above with [`hri_person_manager`](https://github.com/ros4hri/hri_person_manager) and [`libhri`](https://github.com/ros4hri/libhri).

## - Revert the changes introduced with commit https://github.com/ros4hri/pyhri/commit/be95f6a9ed7caf397f532a3d713496451047266e - WORKING
Going back to `sensor_msgs/RegionOfInterest` for the present repo allows completing the walkthrough (i.e. to make the mentioned packages work together).

# TLDR
It looks like that the use of `hri_msgs/NormalizedRegionOfInterest2D` has been introduced for this repo, but not for other ROS4HRI packages. 
Though this is can be solved by going back to `sensor_msgs/RegionOfInterest` messages, I guess that a cleaner idea could be to adapt all the ROS4HRI packages to use `hri_msgs/NormalizedRegionOfInterest2D`. 
While working on this, keeping `master` branches compatible with each other (and thus with the tutorial) would be a nice thing for newcomers!

**Comment by LorenzoFerriniCodes** [comment]
Hi @Luca-Pozzi! I'm happy you're appreciating our work. 

We'll deal with the problem asap. The REP-155 (that is, the one defining the ROS4HRI standard) does not mention NormalizedRegionOfInterest2D messages, so sensor_msgs/RegionOfInterest is actually the message type pythri should be using.

**Comment by Luca-Pozzi** [comment]
Hi @LorenzoFerriniCodes,
I have just proposed a pull request to go back to `sensor_msgs/RegionOfInterest` messages. 
If useful, I am happy to collaborate!
